diff --git a/src/ulib.props b/src/ulib.props
index f323770..5bab4ba 100644
--- a/src/ulib.props
+++ b/src/ulib.props
@@ -7,14 +7,14 @@
     <DotPath>C:\Projekte\Tools\dot\bin\</DotPath>
   </PropertyGroup>
   <PropertyGroup>
-    <OutDir>$(SolutionDir)..\bin\$(Configuration)\$(PlatformName)\$(ProjectName)\</OutDir>
-    <IntDir>$(SolutionDir)..\intermediate\$(Configuration)\$(PlatformName)\$(ProjectName)\</IntDir>
+    <OutDir>$(MSBuildThisFileDirectory)..\bin\$(Configuration)\$(PlatformName)\$(ProjectName)\</OutDir>
+    <IntDir>$(MSBuildThisFileDirectory)..\intermediate\$(Configuration)\$(PlatformName)\$(ProjectName)\</IntDir>
   </PropertyGroup>
   <ItemDefinitionGroup>
     <ClCompile>
       <WarningLevel>Level4</WarningLevel>
       <SDLCheck>true</SDLCheck>
-      <AdditionalIncludeDirectories>$(SolutionDir)include;$(ProjectDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)include;$(ProjectDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>BOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <LanguageStandard>stdcpp17</LanguageStandard>
       <DisableSpecificWarnings>26812</DisableSpecificWarnings>
